### DISCUSSION TITLE
Line/bar distribution plot (ECharts histogram)

### DISCUSSION THREAD

**Discussion by shuaiwang88** [apache/superset#24215] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'd like to duplicate the following code in Seaborn, where I can plot a histogram-like density plot, without the histogram bar. 
x = np.random.randn(100)
ax = sns.distplot(x, hist=False, kde = True)

**Describe the solution you'd like**
Something like this, group by a level.
<img width="975" alt="Screen Shot 2021-09-08 at 9 50 53 PM" src="https://user-images.githubusercontent.com/14266357/132608911-f95872fe-2341-4b50-857f-0b4bcfabef71.png">

**Describe alternatives you've considered**

**Additional context**

**Comment by ValentinC-BR** [comment]
This is definitley something we need.

**Comment by ClaraR-BR** [comment]
We need this feature.

**Comment by rusackas** [comment]
I like the idea! Like mentioned in [this thread](https://github.com/apache/superset/issues/16743), I believe we need to create an ECharts-based histogram plugin to replace the legacy one anyway. It could be one plugin that does both lines and bars, or two different plugins, but they'd be pretty similar in scope, I think. 

This isn't a bug, but I think it's a worthwhile Idea, so I'll convert it to an Ideas thread for now, and hope it garners some interest.

**Comment by rusackas** [comment]
Per https://github.com/apache/superset/issues/16743, the legacy Histogram doesn't support clickable legends. The new chart should.

**Comment by rusackas** [comment]
Another request we've heard several times now is that like the ECharts line/bar plot, people will want it to work in horizontal AND vertical layout mode.

**Comment by rusackas** [comment]
Make sure tooltips work according to best practices. See this prior feature request, which I've closed to include here instead. https://github.com/apache/superset/discussions/18424

**Comment by rusackas** [comment]
I took the liberty of giving this a broader title since it's becoming the hub ticket for echarts histogram requests.

**Comment by rusackas** [comment]
Add a dynamic/configurable number of bins: https://github.com/apache/superset/issues/21039
